src/Time/Actions/ChangeStepSize.hpp

```diff
@@ -90,6 +93,35 @@ bool change_step_size(const gsl::not_null<db::DataBox<DbTags>*> box) {
     can_change_step_size =
         time_stepper.can_change_step_size(time_step_id, history);
   });
+
+  const auto current_step = db::get<Tags::TimeStep>(*box);
+
+  std::optional<size_t> fixed_lts_ratio{};
+  if constexpr (db::tag_is_retrievable_v<Tags::FixedLtsRatio,
+                                         db::DataBox<DbTags>>) {
+    fixed_lts_ratio = db::get<Tags::FixedLtsRatio>(*box);
+  }
+
+  TimeStepRequestProcessor step_requests(time_step_id.time_runs_forward());
+  bool step_accepted = true;
+  if (fixed_lts_ratio.has_value()) {
+    ASSERT(*fixed_lts_ratio > 0 and
```

Comment:
[optional] I think we've been preferring `.value()` on optionals to make it a bit easier to read